src/SDKs/Kusto/Management.Kusto/Generated/SdkInfo_KustoManagementClient.cs

```diff
@@ -0,0 +1,41 @@
+
+// <auto-generated>
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Management.Kusto
+{
+  using System;
+  using System.Collections.Generic;
+  using System.Linq;
+
+  internal static partial class SdkInfo
+  {
+      public static IEnumerable<Tuple<string, string, string>> ApiInfo_KustoManagementClient
+      {
+          get
+          {
+              return new Tuple<string, string, string>[]
+              {
+                new Tuple<string, string, string>("Kusto", "Clusters", "2018-09-07-preview"),
```

Comment:
If the REST API version is preview, the Nuget packages published must also be preview. Please start the nuget package versioning from `0.9.0-preview`